Gemfile

```diff
@@ -23,7 +24,14 @@ group :development, :test do
   gem 'puppetlabs_spec_helper',  :require => false
   gem 'puppet-lint',             :require => false
   gem 'simplecov',               :require => false
-  gem 'beaker-rspec', *location_for(beaker_rspec_version)
+  if beaker_version
+    gem 'beaker', *location_for(beaker_version)
+  end
+  if beaker_rspec_version
+    gem 'beaker-rspec', *location_for(beaker_rspec_version)
+  else
+    gem 'beaker-rspec',  :require => false
+  end
```

Comment:
This PR doesn't change the exposure of Windows to beaker-rspec. ie it was installed before, and installation will still be attempted after this PR, it will just use more of Bundler's options when installing it. If the Gemfile didn't work before, it still won't work, if it worked before, it should continue to work.